web-console/lib/keywords.js

```diff
@@ -34,6 +34,10 @@ exports.SQL_KEYWORDS = [
   'DESC',
   'LIMIT',
   'UNION ALL',
+  'JOIN',
+  'LEFT',
+  'INNER',
+  'ON',
```

Comment:
What about RIGHT? Are there any other keywords that are now supported that should go here?